bundles/org.openhab.voice.porcupineks/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.voice.porcupineks</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Porcupine Keyword Spotter</name>
+  <dependencies>
+    <dependency>
+      <groupId>ai.picovoice</groupId>
+      <artifactId>porcupine-java</artifactId>
+      <version>2.0.2</version>
+    </dependency>
+  </dependencies>
```

Comment:
To be checked by someone if this requires something in the feature.xml file.

Reply:
Reading documentation here https://www.openhab.org/docs/developer/buildsystem.html
I understand that in your case you have nothing to add in your feature.xml file (so your feature.xml is fine) but I also understand that you should add `<scope>compile</scope>` in your POM file.

@wborn or @J-N-K : please confirm my assumptions.

Reply:
Correct. It‘ll probably work without the scope since `compile` is the default scope, but should be added for clarity. Nothing needs to be added to the feature if the dependency is embedded.

Reply:
Thank you for the explanation.

Reply:
Thank you, I'll add the compile scope for clarity.